cellular_automata/game_of_life.py

```diff
@@ -98,7 +98,7 @@ def __judge_point(pt: bool, neighbours: list[list[bool]]) -> bool:
     if pt:
         if alive < 2:
             state = False
-        elif alive == 2 or alive == 3:
+        elif alive in (2, 3):
```

Comment:
As discussed in `ruff rule PLR1714`
> Use a `set` if the elements are washable.

```suggestion
        elif alive in {2, 3}:
```
Repeat below.